daemon/health.go

```diff
@@ -133,12 +133,19 @@ func (p *cmdProbe) run(ctx context.Context, d *Daemon, cntr *container.Container
 	case <-tm.C:
 		cancelProbe()
 		logrus.WithContext(ctx).Debugf("Health check for container %s taking too long", cntr.ID)
-		// Wait for probe to exit (it might take a while to respond to the TERM
-		// signal and we don't want dying probes to pile up).
+		// Wait for probe to exit (it might take some time to call containerd to kill
+		// the process and we don't want dying probes to pile up).
 		<-execErr
+		out := output.String()
+		var msg string
+		if len(out) > 0 {
```

Comment:
Oh, since you need to update for the other bit; perhaps also make the `out` variable local here (slightly clearer to indicate it's not used elsewhere);
```suggestion

		var msg string
		if out := output.String(); len(out) > 0 {
```

Reply:
👇 oh, and perhaps you have thoughts on the newline as well; I'm not _really_ tried to it, mostly wondering if using a space instead of a newline would be more "standard" (and possibly ease grep-ing for failed checks combined with output - not sure)